tests/unit/drivers/observability/test_open_telemetry_observability_driver.py

```diff
@@ -163,6 +163,8 @@ def test_observability_agent(self, driver, mock_span_exporter):
                 ExpectedSpan(name="Agent.run()", parent="main", status_code=StatusCode.OK),
                 ExpectedSpan(name="Agent.before_run()", parent="Agent.run()", status_code=StatusCode.OK),
                 ExpectedSpan(name="Agent.try_run()", parent="Agent.run()", status_code=StatusCode.OK),
+                ExpectedSpan(name="PromptTask.before_run()", parent="Agent.try_run()", status_code=StatusCode.OK),
+                ExpectedSpan(name="PromptTask.after_run()", parent="Agent.try_run()", status_code=StatusCode.OK),
```

Comment:
Well, this gives me some extra confidence that the move of `@observable` kept the same functionality.

Thank you unit tests 🙏 .